torch/_inductor/pattern_matcher.py

```diff
@@ -1356,9 +1356,7 @@ def fwd_only(fn, args, *, run_dce=True) -> torch.fx.GraphModule:
     """Build a normalized inference graph, for use with fx_to_pattern"""
     # TODO - look into using aot autograd, asserting no mutating ops here
     with enable_python_dispatcher():
-        mode = (
-            "real" if not torch._inductor.utils.any_is_symbolic(*args) else "symbolic"
-        )
+        mode = "real"
```

Comment:
Interestingly we were doing the exact opposite of your suggestion @ezyang.  This change does fix it, though let's see what CI thinks as I'm not sure what is up with the prior code.